worker/src/handles/TcpConnection.cpp

```diff
@@ -332,7 +332,7 @@ void TcpConnection::Write(const uint8_t* data1, size_t len1, const uint8_t* data
 
 void TcpConnection::ErrorReceiving()
 {
-	MS_ERROR();
+	MS_ERROR("");
```

Comment:
Please, replace this with `MS_TRACE()`. It was a typo.

Reply:
Don't forget to replace it, please :)

Reply:
already merged the  'fix typo' commit from current v3 branch.